uint8   id
uint8   series
uint16  cw_angle_limit
uint16  ccw_angle_limit
uint8   cw_compliance_margin
uint8   ccw_compliance_margin
uint8   cw_compliance_slope
uint8   ccw_compliance_slope
uint8   p_gain
uint8   i_gain
uint8   d_gain
uint16  torque_limit
uint16  punch

# Valid Dynamixel Series for "series" parameter
uint8   AX = 1
uint8   MX64 = 2
